.github/workflows/ci.yaml

```diff
@@ -1,25 +0,0 @@
-name: CI build
```

Comment:
Please restore this file

Reply:
sorry, think it was deleted when I imported to the 2022 wpilib somehow

Reply:
Yea, makes sense. No worries.